dev-requirements.txt

```diff
@@ -1,9 +1,9 @@
 bravado_core==5.0.7
 flake8==3.6.0
-pytest==3.9.3
+pytest==3.10.0
 pytest-cache==1.0
 pytest-cov==2.6.0
-pytest-sugar==0.9.1
+pytest-sugar-bugfix159==1.0
```

Comment:
I am really reluctant of merging that because then we might loose sight of the next release of `pytest-sugar`.